From 042b7a77df022f8d0d36e490467277d67d1bb3e1 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Fri, 13 Jul 2018 10:00:19 +0800
Subject: [PATCH 1338/1345] arm64: dts: marvell: add timeout-ms property for
 armada platform.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add timeout-ms property in i2c dts node of cp110-master and cp110-slave
configure space.

This patch refer to SDK commit ID d52ede3f8143(“dts: a8040: introduce
initial DTS file hierarchy for Armada-8040”) and commit ID b4f0dd69f759
(“dts: cp110: i2c: added description of 2 I2C interfaces”)

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-cp110-master.dtsi      |    2 ++
 .../arm64/boot/dts/marvell/armada-cp110-slave.dtsi |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
index b5ab005..0218603 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
@@ -230,6 +230,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				interrupts = <ICU_GRP_NSR 120 IRQ_TYPE_LEVEL_HIGH>;
+				timeout-ms = <1000>;
 				clocks = <&cpm_syscon0 1 21>;
 				status = "disabled";
 			};
@@ -240,6 +241,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				interrupts = <ICU_GRP_NSR 121 IRQ_TYPE_LEVEL_HIGH>;
+				timeout-ms = <1000>;
 				clocks = <&cpm_syscon0 1 21>;
 				status = "disabled";
 			};
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
index 12e3053..6185ac8 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
@@ -191,6 +191,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				interrupts = <ICU_GRP_NSR 120 IRQ_TYPE_LEVEL_HIGH>;
+				timeout-ms = <1000>;
 				clocks = <&cps_syscon0 1 21>;
 				status = "disabled";
 			};
@@ -201,6 +202,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				interrupts = <ICU_GRP_NSR 120 IRQ_TYPE_LEVEL_HIGH>;
+				timeout-ms = <1000>;
 				clocks = <&cps_syscon0 1 21>;
 				status = "disabled";
 			};
-- 
1.7.9.5

